bridge/client/app/_interceptors/http-error-interceptor.ts

```diff
@@ -22,24 +22,20 @@ export class HttpErrorInterceptor implements HttpInterceptor {
               @Inject(RETRY_ON_HTTP_ERROR) private hasRetry: boolean) {
   }
 
-  // tslint:disable-next-line:no-any
-  intercept(request: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
-    const params = this.hasRetry ? undefined : {maxAttempts: 3, scalingDuration: 0, shouldRetry: () => false};
+  intercept(request: HttpRequest<unknown>, next: HttpHandler): Observable<HttpEvent<unknown>> {
+    const params: RetryParams | undefined = this.hasRetry ? undefined : {maxAttempts: 3, scalingDuration: 0, shouldRetry: () => false};
     return next.handle(request)
       .pipe(
         retryWhen(genericRetryStrategy(params)),
         catchError((error: HttpErrorResponse) => {
-
           if (error.status === 401) {
             this._handleUnauthorizedError(error);
           } else if (error.error instanceof ErrorEvent) {
             // A client-side or network error occurred. Handle it accordingly.
-            console.error('An error occurred:', error.error.message);
             this.toast.create(`${error.error.message}`);
```

Comment:
The toast disappears after a few seconds and the `console.error` allows to find the error in the console logs again. Not sure if we should remove this? I would actually keep it.

Reply:
I don't think that we log something internal here just an API response. For debugging and traceablity I would also suggest to keep it.

Reply:
After some investigation, we found out that the browser logs failed HTTP calls anyway and since we don't provide any additional details to the user, we can skip this `console.error`.

For our refactoring we should definitly consider to provide better error messages to the user, like "`Project xyz could not be found`", instead of "`HTTP GET /project/xyz failed with 404`".

@laneli @thisthat can you add this to our refactoring task?